src/reduction.jl

```diff
@@ -172,14 +172,15 @@ function controller_reduction(P::ExtendedStateSpace, K, r, out=true; kwargs...)
 end
 
 """
-    hsvd(sys::AbstractStateSpace{Continuous})
+    hsvd(sys::AbstractStateSpace)
 
 Return the Hankel singular values of `sys`, computed as the eigenvalues of `QP`
 Where `Q` and `P` are the Gramians of `sys`.
 """
-function hsvd(sys::AbstractStateSpace{Continuous})
-    P = MatrixEquations.plyapc(sys.A, sys.B)
-    Q = MatrixEquations.plyapc(sys.A', sys.C')
+function hsvd(sys::AbstractStateSpace)
+    fun = ControlSystems.isdiscrete(sys) ? MatrixEquations.plyapd : MatrixEquations.plyapd
```

Comment:
Why not `fun = Matrixequations.plyapd` ?

Reply:
Thanks for noticing 
https://github.com/JuliaControl/RobustAndOptimalControl.jl/commit/06454ea0519ffe328eb415640b99f7dc8e3d31d0